extension/models/requestProducts/simpleProduct.js

```diff
@@ -2,13 +2,23 @@ class SimpleProduct {
   constructor (productId, quantity) {
     this.productId = productId
     this.quantity = quantity
+    this.properties = {}
+  }
+
+  /**
+   * @param {string} key
+   * @param {string} value
+   */
+  addProperties (key, value) {
```

Comment:
because we are handling the properties here
https://developer.shopgate.com/references/connect/shopgate-pipelines/cart/product/shopgate.cart.addproducts.v1
It's just called properties on the connect side.

Reply:
Yes, I understand that Shopgate uses different terminology, but this class is responsible for building a request to magento, magento calls it options, to avoid developer confusion (I was when reading it), this should not be properties. In the return it also identifies the issue when you are doing `options: this.properties`.